localstack/services/cloudformation/models/ec2.py

```diff
@@ -577,10 +595,6 @@ def _store_instance_id(result, resource_id, resources, resource_type):
             },
             "delete": {
                 "function": "terminate_instances",
-                "parameters": {
-                    "InstanceIds": lambda params, **kw: [
-                        kw["resources"][kw["resource_id"]]["PhysicalResourceId"]
-                    ]
-                },
+                "parameters": {"InstanceIds": ["InstanceId"]},
```

Comment:
nit: If I'm not mistaken, this would create a parameters dict like 
```
{"InstanceIds": "<my-instance-id>"}
```
instead of
```
{"InstanceIds": ["<my-instance-id>"]}
```

Not very intuitive, but that's the current logic in `template_deployer.py`, if I'm reading it correctly: https://github.com/localstack/localstack/blob/master/localstack/services/cloudformation/engine/template_deployer.py#L920-L922

Reply:
Will fix in a follow-up soon :+1: